modules/client-configuration/pages/contact-methods-salt.adoc

```diff
@@ -1,8 +1,29 @@
 [[contact-methods-salt]]
-= Contact Methods for Salt Clients
+= Salt Minion Configuration File
 
-Salt clients can communicate with the {productname} Server using SSH.
+In most cases, Salt clients are registered accurately with the default bootstrap methods.
+However, you can use Salt to register the client to the {productname} Server manually by editing the Salt minion file on the client, and providing the fully qualified domain name (FQDN) of the server.
+This method operates using port 4505 outbound on the server and port 4506 inbound.
```

Comment:
@Loquacity Sorry that I'm so late to the party, but this is not correct. Both connections are initiated by the minion to the master: https://docs.saltstack.com/en/getstarted/system/communication.html

The relevant quotes:
`Publisher | (port 4505) All Salt minions establish a persistent connection to the publisher port…`
and
`Request Server | (port 4506) Salt minions connect to the request server as needed to send results to the Salt master…`

Reply:
Thanks for letting me know! New PR here: https://github.com/uyuni-project/uyuni-docs/pull/639